out/production/hotel-booking-currency-service/templates/checkout.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <title>Title</title>
+</head>
+<body>
+<form action='/charge' method='POST' id='checkout-form'>
+    <input type='hidden' th:value='${amount}' name='amount'/>
+    <label>Price:<span th:text='${amount / 100}'/></label>
+    <!-- NOTE: data-key/data-amount/data-currency will be rendered by Thymeleaf -->
```

Comment:
Sytleguide!!!! (Remove comments, use the right indentation, and i know it sounds awesome, but get the f**k out from that data-name )